app/src/main/java/ru/semper_viventem/pixel4scanner/MainActivity.java

```diff
@@ -391,7 +394,7 @@ private Bitmap processDepthMap(Frame frame) {
                 short rawDepth = (short) (y16 & 8191);
                 float scaledDepth = 1.0f - Floats.constrainToRange(((float) rawDepth) / MAX_VISIBLE_DEPTH_MM, f, 1.0f);
                 float depth = getDepthFromRawData(rawDepth);
-                if ((depth > 0.6f && depth < 0.3f && visualizationMode == VisualizationMode.RENDER_DEPTH_MAP_WITH) || depth > 0.8f) {
+                if (depth>=1.0f) { //(depth > 0.6f && depth < 0.3f && visualizationMode == VisualizationMode.RENDER_DEPTH_MAP_WITH) || depth > 0.8f) {
```

Comment:
Are you sure that you need of this change? I added this condition to filter points that are too close and too far away when building a depth map. This is an easy way to remove fluctuations.